docs/products/redis/howto/warning-overcommit_memory.rst

```diff
@@ -0,0 +1,8 @@
+Handle warning ``overcommit_memory``
+====================================
+
+When starting a Redis service on `Aiven console <https://console.aiven.io/>`_, you may notice on **Logs** the following **warning** ``overcommit_memory``::
+
+    # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
+
+This warning can be safely ignored as the available memory will never drop low enough to hit this particular failure case.
```

Comment:
I feel that the part of the initial explanation why it is so still worth keeping :) in the original it says that "we limit Redis memory usage a little differently from how the software is usually deployed", we can keep it as "Aiven for Redis ensures that the available memory will never drop..."? 

What do you think?

Reply:
I think it makes sense cause still saves the initial thought on it :)

Reply:
Like this: https://github.com/aiven/devportal/pull/434/commits/c78f1c2e29a963535abd57b6649100485f0770ce
@anelook ^

Reply:
Yes! Exactly :) Thx for changing it!